torchgeo/trainers/change.py

```diff
@@ -0,0 +1,303 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Trainers for change detection."""
+
+import os
+import warnings
+from typing import Any, Optional, Union, cast
+
+import segmentation_models_pytorch as smp
+import torch
+import torch.nn as nn
+from torch import Tensor
+from torchmetrics import MetricCollection
+from torchmetrics.classification import (
+    MulticlassAccuracy,
+    MulticlassF1Score,
+    MulticlassJaccardIndex,
+)
+from torchmetrics.wrappers import ClasswiseWrapper
+from torchvision.models._api import WeightsEnum
+
+from ..models import FCSiamConc, FCSiamDiff, get_weight
+from . import utils
+from .base import BaseTask
+
+
+class FocalJaccardLoss(nn.Module):
```

Comment:
Should go in `torchgeo/losses/` ?

Reply:
That makes sense, I'll put this and `XEntJaccardLoss` there.